### ISSUE TITLE
Support Bazel 1.0.0

### ISSUE THREAD

**Issue by geaden** [jin/rules_experimental_android#5]
Hi @jin!

According to this CL https://github.com/bazelbuild/bazel/commit/12fc87cb670f93b09f50e3f4637e1a4fa740b111#diff-2367017fd30c2980c20b8b44bb202166 `AndroidInstrumentationInfo` doesn't provide `targetApk` and `instrumentationApk` anymore, which are required by `composer`. Any ideas on where to get `instrumentationApk` from? 

By the way, I had to add these lines:

```python
export_files([
  ...
  'tools/bin/avdmanager',
  'tools/emulator'
])
```

to `external/androidsdk/BUILD.bazel` in order to pass target analysis. Is there a way to avoid this?

**Comment by jin** [comment]
Hey, could you file this in bazelbuild/rules_android or bazelbuild/bazel? Either works. I'll pass it on the the change author.

**Comment by geaden** [comment]
I am not 100% sure if it's related to `rule_android` or `bazel` itself. I managed to obtain apk paths by

```python
apk = ctx.attr.test_apk[AndroidInstrumentationInfo].target.signed_apk
test_apk = ctx.attr.test_apk[ApkInfo].signed_apk
```

In the end I came up with a custom rule, based on your `composer_instrumentation_test`, to just execute `shell am instrument`. Here is the [gist](https://gist.github.com/geaden/750757d6703d0d0f17d1b8292e8e76e5). Definitely not the most elegant solution, but made android instrumentation testing slightly easier with Bazel on macOS. 

I would apply the patch with apk paths to `composer_instrumentation_test` rule, if you are ok with that.
